File: 2022-2023/sourcecode/lab/lab13/DaggerHiltROOMBasicsJava/app/src/main/java/pl/edu/uwr/pum/dagger_hilt_room_basicsjava/data/Student.java
package pl.edu.uwr.pum.dagger_hilt_room_basicsjava.data;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "student")
public class Student {

    @PrimaryKey(autoGenerate = true)
    private int id;

    private final String name;

    public Student(int id, String name) {
        this.id = id;
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public int getId() {
        return id;
    }
}

File: 2022-2023/sourcecode/lab/lab13/DaggerHiltROOMBasicsKotlin/app/src/main/java/pl/edu/uwr/pum/dagger_hilt_room_basicskotlin/data/Student.kt
package pl.edu.uwr.pum.dagger_hilt_room_basicskotlin.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "student")
data class Student(
    @PrimaryKey(autoGenerate = true) var id: Int = 0,
    val name: String)